holidays/constants.py

```diff
@@ -53,3 +53,4 @@
 HEBREW = "hebrew"
 HINDU = "hindu"
 ISLAMIC = "islamic"
+PROTESTANT = "protestant"
```

Comment:
It would be more logical to add the category CATHOLIC. Then Assumption Day will belong to `by_catholic_holidays`.

Reply:
Hmm, true - my only concern is that it changes the default behavior for `BY` subdivision.

Reply:
Implemented.